### PR TITLE
Test if this class was cause of failure

### LINKED ISSUE #2365: Weird warnings while testing
While testing locally, I'm seeing weird warnings:

```
/Users/ryzokuken/.rvm/gems/ruby-2.3.4/gems/activerecord-4.1.16/lib/active_record/associations/builder/collection_association.rb:38: warning: already initialized constant TagNodeAssociationExtension
/Users/ryzokuken/.rvm/gems/ruby-2.3.4/gems/activerecord-4.1.16/lib/active_record/associations/builder/collection_association.rb:38: warning: previous definition of TagNodeAssociationExtension was here
```

Let's find out why they show up.

/cc @jywarren @publiclab/reviewers

### PR DESCRIPTION
**Pull request by cesswairimu**
Fixes #2365 
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/map/_plainInlineLeaflet.html.erb [hunk 1]
```diff
@@ -11,7 +11,9 @@
     width: auto !important;
   }
   </style>
-  <div class="leaflet-map inline-fullwidth-content" id="map<%= unique_id %>"></div>
+ <%# <div class="leaflet-map inline-fullwidth-content" id="map<%= unique_id %1>"></div> %>
```
Comment (cesswairimu) [inline app/views/map/_plainInlineLeaflet.html.erb]: Hey @jywarren, any ideas why this class is causing travis to fail? these lines 
```
FAIL["test_that_NodeShared_can_be_used_to_convert_short_codes_like_[map:content:lat:lon]_into_maps_which_display_notes", #<Minitest::Reporters::Suite:0x000055c4d437d5b8 @name="NodeSharedTest">, 7.643299305000028]
 test_that_NodeShared_can_be_used_to_convert_short_codes_like_[map:content:lat:lon]_into_maps_which_display_notes#NodeSharedTest (7.64s)
        Expected: 1
          Actual: 0
        test/unit/node_shared_test.rb:153:in `block in <class:NodeSharedTest>'
 FAIL["test_that_NodeShared_can_be_used_to_convert_short_codes_like_[map:tag:blog:lat:lon]_into_maps_which_display_notes,_but_only_those_tagged_with_\"blog\"", #<Minitest::Reporters::Suite:0x000055c4d41fecc8 @name="NodeSharedTest">, 7.680251494999993]
 test_that_NodeShared_can_be_used_to_convert_short_codes_like_[map:tag:blog:lat:lon]_into_maps_which_display_notes,_but_only_those_tagged_with_"blog"#NodeSharedTest (7.68s)
        Expected: 1
          Actual: 0
        test/unit/node_shared_test.rb:160:in `block in <class:NodeSharedTest>'
```

**Comment by jywarren** [comment]
What if we wrap the Leaflet div in a second div and have each one have a different class? the inner one should be the leaflet-map one because i think it may be filled by Leaflet and its contents replaced. Want to give this a try? The `inline-fullwidth-content` would then be the outer div.

**Comment by cesswairimu** [comment]
Aha, nice trying that pushing up..

**Comment by cesswairimu** [comment]
@jywarren  wrapping the div outside works  what page do I need  to check to see if the style is fine

**Comment by jywarren** [comment]
uh, to add an inline layers map as in https://publiclab.org/wiki/inline-maps thanks!!

**Comment by jywarren** [comment]
Ah ok I tested this in the developer console by directly editing the page, and it worked!

**Pull request merged**